cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/3.0/casServiceValidationSuccess.jsp

```diff
@@ -49,9 +49,6 @@
                         <cas:${fn:escapeXml(attr.key)}>${fn:escapeXml(attrval)}</cas:${fn:escapeXml(attr.key)}>
                     </c:forEach>
                 </c:forEach>
-                <cas:isFromNewLogin>${fn:escapeXml(isFromNewLogin)}</cas:isFromNewLogin>
```

Comment:
What you have in mind seem like client-specific to me. For instance, a client could do:

`getCasAssertion().getAuthenticationAttributes()...`

or 

`getCasAssertion().getPrincipalAttributes()...`

...which leaves/allows the client to do all the parsing needed and make that separation, and decouples the CAS server from any future changes.